aten/src/ATen/native/cpu/WeightNormKernel.cpp

```diff
@@ -332,6 +332,11 @@ void weight_norm_backward_last_dim_kernel(
   Tensor buffer = at::empty({num_threads, N}, saved_norm.options()).zero_();
   auto buffer_data = buffer.data_ptr<accscalar_t>();
 
+  // 1st row to store coefficient a and 2nd row to store coefficient b
+  Tensor a_b = at::empty({2, N}, saved_norm.options()).zero_();
```

Comment:
Hmm, wouldn't be more convenient to just set 1st dimension of buffer tensor to `std::max(2, N)`

Reply:
Good idea! Updated!